ui/client/components/ViewFeatures.js

```diff
@@ -44,7 +45,7 @@ export const ConfidenceBar = withStyles((theme) => ({
 }))(LinearProgress);
 
 const semanticSearchFeatures = async(query) => {
-  let url = `/api/dojo/features/search?query=${query}`;
+  let url = `/api/dojo/features/search?query=${query}&size=50`;
```

Comment:
I'm not sure why this was 10. On document search, I had bumped it some. Adding more results to allow for both:
a) Actual keyword/fuzzy sentence results first
b) Semantic results afterwards to fill up to 50 results